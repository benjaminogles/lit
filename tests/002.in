
lit-meta
title Test Meta
author Benjamin Ogles
description A test for lit: a literate programming tool

lit-end

This content should be ignored
title should be ignored
author should be ignored
descrition should be ignored

lit-meta
no matching keys on this line
url https://github.com/benjaminogles

image image.png

lit

This content should be ignored
title should be ignored
author should be ignored
descrition should be ignored

// lit-meta
// description Embedding site metadata in source code comments
// 
