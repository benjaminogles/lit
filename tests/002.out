<!doctype html>

<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
<title>Test Meta</title>
<meta property="og:title" content="Test Meta">
<meta name="author" content="Benjamin Ogles">
<meta name="description" content="A test for lit: a literate programming tool">
<meta property="og:description" content="A test for lit: a literate programming tool">
<meta property="og:url" content="https://github.com/benjaminogles">
<meta property="og:image" content="image.png">
<meta name="description" content="Embedding site metadata in source code comments">
<meta property="og:description" content="Embedding site metadata in source code comments">
</head>
<body>
<main>
<article>
</article>
</main>
</body>
</html>
